test/Microsoft.Azure.Devices.Edge.Test.Common/DaemonConfiguration.cs

```diff
@@ -39,13 +40,28 @@ public void SetDpsSymmetricKey(string idScope, string registrationId, string dev
         {
             this.config.RemoveIfExists("provisioning");
             this.config.ReplaceOrAdd("provisioning.source", "dps");
-            this.config.ReplaceOrAdd("provisioning.global_endpoint", "https://global.azure-devices-provisioning.net");
+            this.config.ReplaceOrAdd("provisioning.global_endpoint", GlobalEndPoint);
             this.config.ReplaceOrAdd("provisioning.scope_id", idScope);
             this.config.ReplaceOrAdd("provisioning.attestation.method", "symmetric_key");
             this.config.ReplaceOrAdd("provisioning.attestation.registration_id", registrationId);
             this.config.ReplaceOrAdd("provisioning.attestation.symmetric_key", deviceKey);
         }
 
+        public void SetDpsX509(string idScope, string registrationId, CertificateAuthority ca)
```

Comment:
This method should take a certificate object directly instead of `CertificateAuthority`.